Table/Base.php

```diff
@@ -434,6 +434,28 @@ public function setPaginator(Paginator\BaseSource $paginator)
         $this->_paginator = $paginator;
     }
 
+    /**
+     * Return the table's Zend paginator.
+     *
+     * @return string
```

Comment:
I don't believe this method returns a string

Reply:
Updated